### PR TITLE
feat: fetch known wallets from repository

### PR DESCRIPTION
**Pull request by dated**
## Summary

Aligns the multisignature participants to the right and fetches the known wallets from https://github.com/ArkEcosystem/common.

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

**Comment by ghost** [comment]
**Thanks for submitting this pull request!** A maintainer will review this in the next few days and explicitly select labels so you know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

**Comment by ItsANameToo** [comment]
@dated checkout is messed up again

**Review by ItsANameToo — DISMISSED** [review]

**Comment by ghost** [comment]
A member has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Comment by ItsANameToo** [comment]
Running tests locally, will merge once those are done

**Comment by ItsANameToo** [comment]
@dated can you update the homepage e2e search test? It searches for a known wallet but that one has been renamed now so the addresses no longer match up.

File: __tests__/e2e/specs/homepage_spec.js [hunk 1]
```diff
@@ -352,13 +352,13 @@ describe("Homepage", () => {
...
           .first()
-          .type("ARK Bounty{enter}");
+          .type("Genesis Wallet{enter}");
 
         cy.get("h1").should($heading2 => {
           expect($heading2.text()).not.to.eq(heading);
         });
 
-        cy.url().should("include", "/wallets/AYCTHSZionfGoQsRnv5gECEuFWcZXS38gs");
+        cy.url().should("include", "/wallets/AewxfHQobSc49a4radHp74JZCGP8LRe4xA);
```
Comment (ItsANameToo) [inline __tests__/e2e/specs/homepage_spec.js]: missing quote at the end

**Review by ItsANameToo — APPROVED** [review]

**Comment by ghost** [comment]
A member has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Comment by ghost** [comment]
Your pull request has been merged and marked as tier 4. _It will earn you $20 USD._

**Pull request merged**